app/routes/labs.missions/route.tsx

```diff
@@ -0,0 +1,95 @@
+import {
+  Grid,
+  GridContainer,
+  Identifier,
+  IdentifierIdentity,
+  IdentifierLogos,
+  IdentifierMasthead,
+} from '@trussworks/react-uswds'
+
+import fermiHeroImage from './Fermi_Earth_GWs.jpg'
+import type { BreadcrumbHandle } from '~/root/Title'
+
+import fleet from './AstroFleetChart.png'
+import meatball from '~/components/meatball/logo.svg'
+
+export const handle: BreadcrumbHandle = {
+  breadcrumb: 'Missions',
+}
+
+export default function () {
+  return (
+    <>
+      <img
+        alt="Fermi satellite with a GRB in the background, both positioned over earth"
+        src={fermiHeroImage}
+      />
+      <GridContainer>
+        <Grid row className="border-bottom border-gray-50">
+          <h1>Mutimessenger missions</h1>
+        </Grid>
+        <Grid row>
```

Comment:
What are these grid rows doing for us? This isn't particularly grid-like; the sections are just appearing from top to bottom on the page.

Reply:
It's getting us centered content without having to use css.
Here's a picture of what it looks like with no grids and no custom css.
<img width="1698" alt="Screenshot 2023-12-11 at 9 23 30 AM" src="https://github.com/nasa-gcn/gcn.nasa.gov/assets/6545874/806e9975-6c30-472b-b7c6-ecac75e7b1c2">

Reply:
On most layout routes we have the following snippet to constrain the content to a centered vertical column with sensible margins:

```jsx
<GridContainer className="usa-section">
  ...
</GridContainer>
```

Try that, enclosing your content but not the hero image, and no `<Grid>` elements beneath it.